.env

```diff
@@ -3,4 +3,6 @@ SEARCH_URL=elasticsearch7://127.0.0.1:9200/haystack
 VIISP_PROXY_AUTH=https://test.epaslaugos.lt/portal/services/AuthenticationServiceProxy
 VIISP_AUTHORIZE_URL=https://test.epaslaugos.lt/portal/external/services/authentication/v2
 VIISP_PID=VIISP-AUTH-SERVICE-01
-TRANSLATION_CLIENT_ID=u-4ac7ee75-4407-435d-8342-f8c789c9144f
\ No newline at end of file
+TRANSLATION_CLIENT_ID=u-4ac7ee75-4407-435d-8342-f8c789c9144f
+CLIENTS_API_URL=https://get-test.data.gov.lt/auth/clients
+CLIENTS_AUTH_BEARER=Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzUxMiJ9.eyJpc3MiOiJodHRwczovL2V4YW1wbGUuY29tLyIsInN1YiI6Ijg5ZGFmNjgzLTg3MjMtNGU1NC1iODUwLTBiOTI0ZTgyOWY5OSIsImF1ZCI6Ijg5ZGFmNjgzLTg3MjMtNGU1NC1iODUwLTBiOTI0ZTgyOWY5OSIsImlhdCI6MTcwMDczMDE4NSwiZXhwIjoxNzAxNTk0MTg1LCJzY29wZSI6InNwaW50YV9hdXRoX2NsaWVudHMifQ.iiBkmCq4MyXabm93CIx_E4GcSv7qB1blpVG1tTbQg878k578VNhz7XrMBlMouoxQnzJX7xLhfzBoC4l5lS2fMSzhHsBcaQqwCNTxZsOJhXM8eh7cuKPUWls1qIoF-HfRAaYTFpZyTtpdzlW_fq877HetFoCGXYfuYe07TyvN1KFAW0D4oy2JfenshogFJY_g0rN_nwimdY-KPUbP_2dQBwFB-VWAi3FFfp-hYB9ZIRc-BWwKhfP2zHNbAuceBPmm-MhHXhE2HDbMIrVrYJyN1EM30sKJJ_vBOxbcAXPdM2-1XleVWjo6oclAEJKMmjWDs46fnGcP8MIoBz_4t9ch0g
```

Comment:
This file can't be added to version control, especially with the secrets! Please remove this file from version control.